wikimedia/wikipedia-ios
# Add `Other` option to filter notifications with unknown types

**Phabricator:** https://phabricator.wikimedia.org/T306569

### Notes
* This PR adds a new filter named `Other` that captures all notifications with unknown types that were not filtered by all the existing filters. I'll try to add some tests separately later.

### Test Steps
Setup -  Pull the `notifications-performance-testing` branch and cherry-pick the commits into it  - you might need to fix some conflicts (do not push these changes to this branch). 
1. Go to the Notifications center. You will notice several new notifications loading
2. Turn off all filters but the `Other` filter. You should see some notifications of the type "alert"
3. Turn off all filters. You should see the empty state view.
4. Turn on the `Other` filter and some other type. Check that only the generic alerts and the other selected type are showing.
